packages/cli/src/lib/packager/packageManager.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { join as joinPath } from 'path';
+import { existsSync } from 'fs';
+import { findRootSync } from '@manypkg/find-root';
```

Comment:
you probably don't need this package; use the `paths` utility in the cli instead, see for example [here](https://github.com/backstage/backstage/blob/b017c80551c79368e226ed50232504413ec83045/packages/cli/src/commands/new/new.ts#L58) some code that resolves something relative to the monorepo root.

Reply:
that worked, thanks!

Reply:
@freben it turned out when @ourchitectureio was testing with pnpm, the "paths" method does not work. Would you be fine if i went back to `@manypkg/find-root` ? that package is an indirect dependency anyway already.